account-kit/java/signer-sdk/src/main/java/com/alchemy/aa/client/SginerClient.java

```diff
@@ -0,0 +1,230 @@
+package com.alchemy.aa.client;
+
+import com.alchemy.aa.Stamper;
+import com.alchemy.aa.Stamper.Stamp;
+import com.alchemy.aa.client.api.AuthJWT;
+import com.alchemy.aa.client.api.AuthUser;
+import com.alchemy.aa.client.api.AuthUser.WhoAmIRequest;
+import com.alchemy.aa.client.api.GetUser;
+import com.alchemy.aa.client.api.SignRawMessage.SignParamter;
+import com.alchemy.aa.client.api.SignRawMessage.SignedRequest;
+import com.alchemy.aa.client.api.SignRawMessage.SignedResponse;
+import com.alchemy.aa.client.api.SignRawMessage.SigningBody;
+import com.fasterxml.jackson.databind.ObjectMapper;
+import com.fasterxml.jackson.databind.ObjectWriter;
+import com.google.crypto.tink.util.Bytes;
+import com.google.protobuf.InvalidProtocolBufferException;
+import java.net.URI;
+import java.net.http.HttpClient;
+import java.net.http.HttpRequest;
+import java.net.http.HttpResponse;
+import java.security.GeneralSecurityException;
+import java.time.Instant;
+import java.util.Optional;
+import lombok.Builder;
+import lombok.Getter;
+import org.bouncycastle.util.encoders.Hex;
+
+public class SginerClient {
+
+    public class HttpConfig {
+
+        public HttpConfig(String api_key) {
```

Comment:
a dev might specify their own RPC url here in case they want to proxy to a different backend